app/BUILD.bazel

```diff
@@ -245,6 +253,50 @@ ALTERED_VIEWS = [
  )
 for file in ALTERED_VIEWS]
 
+# Source files and genrule targets for databinding_adapters library.
+DATABINDING_ADAPTER_FILES = [
+    "src/main/java/org/oppia/app/databinding/EditTextBindingAdapters.java",
+    "src/main/java/org/oppia/app/databinding/GuidelineBindingAdapters.java",
+    "src/main/java/org/oppia/app/databinding/MarginBindingAdapters.java",
+    "src/main/java/org/oppia/app/databinding/StateAssemblerMarginBindingAdapters.java",
+    "src/main/java/org/oppia/app/databinding/StateAssemblerPaddingBindingAdapters.java",
+    "src/main/java/org/oppia/app/databinding/ViewBindingAdapters.java",
+    "update_src/main/java/org/oppia/app/databinding/DrawableBindingAdapters",
+    "update_src/main/java/org/oppia/app/databinding/ImageViewBindingAdapters",
+    "update_src/main/java/org/oppia/app/databinding/TextViewBindingAdapters",
+]
+
+# TODO(#1617): Remove genrules post-gradle
+# Adapter files altered by genrules to be excluded in the app library.
+ALTERED_DATABINDING_ADAPTERS = [
+    "src/main/java/org/oppia/app/databinding/DrawableBindingAdapters.java",
+    "src/main/java/org/oppia/app/databinding/ImageViewBindingAdapters.java",
+    "src/main/java/org/oppia/app/databinding/TextViewBindingAdapters.java",
+]
+
+'''
+Genrule that alters both resource import statements and class names for adapter files in order to
+account for altered package and file name.
```

Comment:
This documentation is a bit confusing--we're altering the imports to account for the altered package & file name? Can this be made a bit clearer?

Reply:
Fixed!